### ISSUE TITLE
HQ-SAM

### ISSUE THREAD

**Issue by dlangenk** [biigle/magic-sam#25]
There is a finetuned variant of SAM available at https://github.com/SysCV/sam-hq which promises better segmentation masks at basically no cost.

To upgrade to sam-hq, we basically only have to replace the sam package that is installed with pip with sam-hq (also pip compatible) and download the new weights. The front end is completely compatible with the new embeddings (already tested).

Although there is no very big perceived improvement, it might be worth it.

**Comment by mzur** [comment]
The low resolution of the feature representation is probably still the most limiting factor. If you don't see a big improvement in the cases you tested in #9, I don't see an immediate need to implement this :thinking:

**Comment by dlangenk** [comment]
Yes it is. But if we solved that it might be worth to implement it. Creating the pull request will only take 10-40 min.

**Comment by mzur** [comment]
Right. But then I want an example that HQ SAM provides better results than the current implementation :wink:
